app/features-json/middleware/jwt_auth.rb

```diff
@@ -5,7 +5,7 @@
 module FastlaneCI
   # API Middleware responsible of authenticate all the requests that uses it.
   class JwtAuth
-    def initialize(app, encryption_key=nil, protect_endpoints_starting_with="/data")
+    def initialize(app, encryption_key = nil, protect_endpoints_starting_with = "/data")
```

Comment:
instead of `=` we generally default to `:` so they can be used as named params

Reply:
This was intentionally done, so the middleware had minimum configuration and preferred convention, but can be changed if you think so

Reply:
In general we prefer to have named parameters. If there's a reason to keep it this way, that's totally cool. I just don't understand the advantage in not using named parameters here.

Reply:
There's no other reason that try to follow how other Rack middleware's parameter naming worked, so I used unnamed params. But ofc we can change it to named one 😄